app/models/dturn-info.js

```diff
@@ -0,0 +1,53 @@
+// https://github.com/EnCiv/civil-pursuit/issues/196
+
+import { Collection } from '@enciv/mongo-collections'
```

Comment:
Okay, I feel like model/collection names should end with an s,, and be short.  So how about calling this dturns?   Also open to other names.